### PR TITLE
Added ReactionMessage for React User Message with Emoji

### PR DESCRIPTION
**Pull request by amisama**
image

### CODE REVIEW

File: src/Utils/messages.ts [hunk 1]
```diff
@@ -276,7 +277,11 @@ export const generateWAMessageContent = async(
 		}
 	} else if('location' in message) {
 		m.locationMessage = WAProto.LocationMessage.fromObject(message.location)
-	} else if('delete' in message) {
+	}
```
Comment (burstfreeze) [inline src/Utils/messages.ts:280]: Please try to stay within the coding style of the repo.

Check the lint scripts in package.json to verify or fix your style so it matches the rest.

Comment (amisama) [reply src/Utils/messages.ts:280]: sorry I'll fix it

**Comment by adiwajshing** [comment]
Nice. What can be added:
1. firing `messages.update` with the reaction when a reaction message is sent/recv

**Comment by amisama** [comment]
> Nice. What can be added:
> 
> 1. firing `messages.update` with the reaction when a reaction message is sent/recv



> Nice. What can be added:
> 
> 1. firing `messages.update` with the reaction when a reaction message is sent/recv
Its there on message.upsert, should I put it in messages.update too ?

IMG-20220321-WA0005

**Comment by adiwajshing** [comment]
@amisama I meant call update for the original message that received the reaction

**Pull request merged**